cypress/e2e/collaborators.spec.ts

```diff
@@ -160,18 +134,10 @@ describe("collaborators flow", () => {
     it("should not be able to remove the last site member", () => {
       // Act
       // NOTE: Remove all collaborators except the initial admin
-      getCollaboratorsModal()
-        .get(DELETE_COLLABORATOR_BUTTON_SELECTOR)
-        .then((buttons) => {
-          if (buttons.length > 1) {
-            buttons.slice(1).each((idx, button) => {
-              button.click()
-            })
-          }
-        })
+      removeOtherCollaborators()
 
       // Assert
-      cy.get(DELETE_COLLABORATOR_BUTTON_SELECTOR).should("be.disabled")
```

Comment:
i think i'll prefer the export - `DELETE_BUTTON_SELECTOR` but i can specialise it to this context ie, 
```ts
import { DELETE_BUTTON_SELECTOR as DELETE_COLLABORATOR_BUTTON_SELECTOR }
```

the reason for this is because the selector itself is a general selector, and it **can select multiple** delete buttons, not just the collaborator one. for reference, the selector itself is `'button[id^="delete-"]'`

Reply:
updated